official/AnalisiDeiRequisiti/res/sections/requisiti.tex

```diff
@@ -121,7 +121,7 @@ \subsection{Requisiti funzionali}
 
 	%<< end
 
-    R1D 14 & Funzionale \newline Desiderabile & Il sistema permette all'utente di scaricare da \glossaryItem{Terminale} una specifica \glossaryItem{DSL}, in un formato leggibile dall'editor. &  Capitolato \newline  UC-E1 \newline \\ \hline
+    R1D 14 & Funzionale \newline Desiderabile & Il sistema permette all'utente di importare una definizione di DSL esistente scrivibile. &  Capitolato \newline  UC-E1 \newline \\ \hline
```

Comment:
Ma per te @EmanueleC è chiaro? Te @Polpetta non dovresti fare niente. Le ho solo riscritte meglio e ho richiesto una pull request in modo che potessimo stendere una versione comprensibile a tutti.